app/serializers/winning_bid_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class WinningBidSerializer < ActiveModel::Serializer
+  root false
+
+  attributes(
+    :amount,
+    :bidder_id
+  )
+
+  def amount
+    object.amount
+  end
+
+  def bidder_id
+    object.bidder_id
```

Comment:
The current bidder is exposed during active auctions.  Looks like you need to use `veiled_bidder`.